index.js

```diff
@@ -12,7 +12,7 @@ function parse (args, opts) {
   args = tokenizeArgString(args)
   // aliases might have transitive relationships, normalize this.
   var aliases = combineAliases(opts.alias || {})
-  var configuration = loadConfiguration(opts.cwd || path.resolve(path.dirname(__filename), '../'))
+  var configuration = loadConfiguration(opts.cwd || rootPath.path)
```

Comment:
@jameswomack No, it uses https://github.com/sindresorhus/find-up internally. Though I honestly don't see the point of the switch as AFAIK `app-root-path` does the same thing as `pkg-conf` and walks up to the closest package.json.